### PR TITLE
[NL] add support for dates in bar, scatter, ranking tiles

### LINKED ISSUE #2392: NL: Map/ranking/bar/scatter charts should show dates
example:

image

### PR DESCRIPTION
**Pull request by beets**
- fixes #2392 
- also fixed a small error in server error handling output formatting

image

image

image

### CODE REVIEW

File: server/lib/nl/page_config_builder.py [hunk 1]
```diff
@@ -540,6 +543,7 @@ def _ranking_chart_block_pc(column, pri_place: Place, pri_sv: str,
   _set_ranking_tile_spec(attr['ranking_types'], pri_sv, tile.ranking_tile_spec)
   tile.title = _decorate_chart_title(title=sv2thing.name[pri_sv],
```
Comment (pradh) [inline server/lib/nl/page_config_builder.py:544]: nit: does https://github.com/datacommonsorg/website/blob/33a84f6c3b8c9fe0557907f442e0780f46af08e7/server/tests/lib/nl/page_config_builder_test.py#L377 not need fixing?

**Review by pradh — APPROVED** [review]
Thanks for the quick change!

LG from backend code perspective :)

File: server/lib/nl/page_config_builder.py [hunk 2]
```diff
@@ -346,7 +346,9 @@ def _multiple_place_bar_block(column, places: List[Place], svs: List[str],
     orig_title = sv2thing.name[svs[0]]
 
   if len(places) == 1:
-    title = _decorate_chart_title(title=orig_title, place=places[0])
+    title = _decorate_chart_title(title=orig_title,
```
[the remark this answers is not in the document]
Comment (beets) [reply server/lib/nl/page_config_builder.py:349]: added, sorry missed those earlier

File: server/lib/nl/page_config_builder.py (hunk 1 above)
Comment (beets) [reply server/lib/nl/page_config_builder.py:545]: ah yes- done.

File: server/lib/nl/page_config_builder.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (pradh) [reply server/lib/nl/page_config_builder.py:349]: Thanks!  Finally, to confirm its intentional, we're not setting add_date in the else (multi-places) case?

**Review by chejennifer — APPROVED** [review]
nice!

**Pull request merged**